llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -1346,6 +1355,15 @@ class LoopVectorizationCostModel {
     return getTailFoldingStyle() != TailFoldingStyle::None;
   }
 
+  bool useWideActiveLaneMask() const {
```

Comment:
For both of the new functions you've added can you add some brief '///' comments?